### ISSUE TITLE
Lint is failing

### ISSUE THREAD

**Issue by ismell** [FichteFoll/SublimeLinter-contrib-sublime-syntax#4]
```
SublimeLinter: linter.py:931          sublimesyntax: checking lint mode 'background' vs lint reason 'on_user_request'
SublimeLinter: #412 linter.py:948     sublimesyntax: linting 'syntax_test_c.c'
SublimeLinter: sublime-syntax: created temporary file at /usr/local/home/me/.config/sublime-text-3/Packages/.temp-subsyn-lint/syntax_test_jct7ou 
SublimeLinter: sublime-syntax: "syntax_test_c.c" assertions: 1 
SublimeLinter: #412 linter.py:1027    sublimesyntax: output:
  Packages/.temp-subsyn-lint/syntax_test_jct7ou: unable to read file
SublimeLinter: #412 linter.py:1063    sublimesyntax: No match for line: 'Packages/.temp-subsyn-lint/syntax_test_jct7ou: unable to read file'
SublimeLinter: sublime_linter.py:449  Linting 'syntax_test_c.c' took 0.01s
```
SublimeLinter v4.9.4
SublimeLinter-contrib-sublime-syntax v1.1.0
Sublime Text v3192

**Comment by FichteFoll** [comment]
Yeah, it's been broken for a while but I haven't bothered to investigate. I guess now is a goot time, considering there's at least someone else bothered by it.

**Comment by FichteFoll** [comment]
Well, that was a weird one. I found a fix that worked for me locally but I had to rely on a wait because ST became weird (see comment in commit). I suspect it might be related to the addition of mini_diff but I can't remember when the linter stopped working for me anymore. Probably earlier, however.

Let me know if this didn't work out for you.
I also improved error reporting at least for this known error message for now.
